server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAppContainer.js

```diff
@@ -125,46 +129,46 @@ class ProjectActivityAppContainer extends React.PureComponent {
     elem && elem.classList.remove('dashboard-page');
   }
 
-  addCustomEvent = (analysis: string, name: string, category?: string): Promise<*> =>
+  addCustomEvent = (analysis /*: string */, name /*: string */, category /*: ?string */) =>
     api
       .createEvent(analysis, name, category)
       .then(
         ({ analysis, ...event }) =>
           this.mounted && this.setState(actions.addCustomEvent(analysis, event))
       );
 
-  addVersion = (analysis: string, version: string): Promise<*> =>
+  addVersion = (analysis /*: string */, version /*: string */) =>
     this.addCustomEvent(analysis, version, 'VERSION');
 
-  changeEvent = (event: string, name: string): Promise<*> =>
+  changeEvent = (event /*: string */, name /*: string */) =>
     api
       .changeEvent(event, name)
       .then(
         ({ analysis, ...event }) =>
           this.mounted && this.setState(actions.changeEvent(analysis, event))
       );
 
-  deleteAnalysis = (analysis: string): Promise<*> =>
+  deleteAnalysis = (analysis /*: string */) =>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)